OracleTuxedo/salt/samples/bankapp/stop_server.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+# 
+# This script is used to attempt to gracefully shutdown the Tuxedo server before its pod is deleted.
+# It assumes that TUXDIR has been set
+
+export APPDIR=/u01/data/bankapp/
+LOG_FILE="${APPDIR}/stop_server.log"
+
+# shellcheck disable=SC1091
+source "${APPDIR}/bankvar.new"
+
+echo "stop server script kicked off at $(date)." > ${LOG_FILE}
```

Comment:
```suggestion
echo "Server shutdown initiated at $(date)" > $LOG_FILE
```

Using `>` will overwrite anything in any existing file with this line. If you want to append this line to an existing log, use `>>`

Reply:
Normally I would have used '>' to prevent overrun of disk space. But I guess here '>>' is fine since this will be written only for manual runs of the stop script. So won't fill up the disk. Plus good to have the past history of shutdown, say for troubleshooting in the future, if required.